### PR TITLE
extra spacing in installation output

### PR DESCRIPTION
**Pull request by kevinushey**
IMHO the extra spacing makes things a bit cleaner, and matches the spacing we use in "standard" pretty print outputs.

```
> renv::install(c("tidyverse", "dplyr"))
# Downloading packages -------------------------------------------------------
[no downloads required]

# Installing packages into "~/Library/R/arm64/4.3/library" -------------------
The following package(s) will be installed:

- dplyr     [1.1.2]
- tidyverse [2.0.0]

Do you want to proceed? [Y/n]: y

Installing dplyr     ... OK [copied from cache]
Installing tidyverse ... OK [copied from cache]

Successfully installed 2 packages in 0.13 seconds.
```

### CODE REVIEW

File: tests/testthat/_snaps/install.md [hunk 1]
```diff
@@ -44,11 +46,13 @@
       - oatmeal   [1.0.0]
       - toast     [1.0.0]
       
+
```
Comment (hadley) [inline tests/testthat/_snaps/install.md:49]: I think we're getting an extra empty line in non-interactive contexts?

**Review by hadley — APPROVED** [review]

**Comment by kevinushey** [comment]
What do you think of the following?

```
> renv::install("tidyverse", "dplyr")
# Downloading packages -------------------------------------------------------
[no downloads required]

The following package(s) will be installed:

- dplyr     [1.1.2]
- tidyverse [2.0.0]

These packages will be installed into "~/Library/R/arm64/4.3/library".

Do you want to proceed? [Y/n]: y

# Installing packages --------------------------------------------------------

- Installing dplyr     ... OK [copied from cache]
- Installing tidyverse ... OK [copied from cache]

Successfully installed 2 packages in 0.15 seconds.
```

**Comment by kevinushey** [comment]
One more option, if we wanted headers that matched what other r-lib / tidyverse packages use:

```
> renv::install("tidyverse", "dplyr")
-- Downloading packages ------------------------------------------------------
Retrieving 'https://cran.rstudio.com/bin/macosx/big-sur-arm64/contrib/4.3/tidyverse_2.0.0.tgz' ...
	OK [downloaded XXXX bytes in 76 milliseconds]
Retrieving 'https://cran.rstudio.com/bin/macosx/big-sur-arm64/contrib/4.3/dplyr_1.1.2.tgz' ...
	OK [downloaded XXXX bytes in 0.14 seconds]

The following package(s) will be installed:

* dplyr     [1.1.2]
* tidyverse [2.0.0]

These packages will be installed into "~/Library/R/arm64/4.3/library".

-- Installing packages -------------------------------------------------------

* Installing dplyr     ... OK [installed binary and cached]
* Installing tidyverse ... OK [installed binary and cached]

Successfully installed 2 packages in 1.2 seconds.
```

**Comment by kevinushey** [comment]
Ok, I like this treatment:

```
> renv::install("tidyverse", "dplyr", rebuild = TRUE, prompt = TRUE)
# Downloading packages -------------------------------------------------------

- Downloading tidyverse from PPM ...            OK [file is up to date]
- Downloading dplyr from PPM ...                OK [file is up to date]

The following package(s) will be installed:

- dplyr     [1.1.2]
- tidyverse [2.0.0]

These packages will be installed into "~/Library/R/arm64/4.3/library".

# Installing packages --------------------------------------------------------

- Installing dplyr ...                          OK [built from source and cached]
- Installing tidyverse ...                      OK [built from source and cached]

Successfully installed 2 packages in 8.7 seconds.
```

File: R/install.R [hunk 2]
```diff
@@ -336,8 +340,8 @@ renv_install_default <- function(records) {
   state <- renv_restore_state()
   handler <- state$handler
 
-  pkg_width <- max(nchar(map_chr(records, function(x) x$Package)), 1)
-  renv_scope_options(renv.package.width = pkg_width)
+  # width <- max(nchar(map_chr(records, function(x) x$Package)), 1)
```
Comment (hadley) [inline R/install.R:343]: Can delete this now

**Pull request merged**